docs/what-is-regression-games.mdx

```diff
@@ -2,58 +2,47 @@
+Regression Games is making it easy to create automated tests for your game.
 
-![vision](img/vision.png)
+## Highlighted Features
 
-## Current Stage of the Product
+| Feature | Description |
+| --- | --- |
+| [Plug-and-Play Setup](getting-started/installing-regression-games) | Simply install our SDK and add one GameObject to your scene to instantly begin using our tools. Custom integration code is rarely needed to get started. |
+| [Get Started Quickly with Recording and Input Playback](getting-started/creating-your-first-automated-test) | Developers can create functional tests by recording a play-through of their scenario and preforming validations against automated playback. |
+| [Computer Vision Capabilities](guides/interacting-with-ui-using-computer-vision) | In some cases, a bot needs to see the game to understand it. Our CV capabilities, such as OCR and object detection, can extract information from in-game screenshots to help drive behaviors and validate game state during runtime. |
+| [Exploratory Testing](guides/finding-game-breaking-bugs-with-stability-testing) | Run more than just predefined scenarios - exploratory bots such as the Monkey Bot can explore edge cases within your game. |
+| [Composability](core-concepts/bot-sequences/getting-started-with-bot-sequences-and-segments) | One approach is not always adequate for automating a certain test scenario. Our drag and drop tool allows you to stitch together and re-use bot logic across tests. |
+| [Runtime, CI/CD, and Editor Compatibility](core-concepts/running-tests-in-the-unity-test-runner) | Our tools work in the Editor, in builds, and within CI/CD. Because of our integration approach, in most cases you will **not need to restart your game** to iterate on a test scenario. |
+| [Data Exploration and Validation](core-concepts/validation-suites) | Our dashboard allows developers to inspect and test the state of their game in a no-code fashion. |
 
-This product is currently a **release preview**. We are actively talking with studios and game developers to discover the best features and use cases to tackle, and we're making changes frequently.
+## Where do I start?
 
-The team has focused on the foundation of supporting use cases and initial infrastructure to get bots to connect to Unity. The bulk
-of our value will come from the interfaces to build bots, the scale of our infrastructure, the speed of integration, and the products
-built around these bots.
+The best place to start is with our [installation guide](getting-started/installing-regression-games). Once our SDK is added to your project, you can [create your first
+test automation](getting-started/creating-your-first-automated-test).
```

Comment:
I interpret "test automation" as the practice, and an "automated test" as a concrete test case to run.

```suggestion
automated test](getting-started/creating-your-first-automated-test).
```